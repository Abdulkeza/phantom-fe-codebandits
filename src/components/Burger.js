import React from 'react';

const Burger = () => {
  return (
    <div>
      <div className="h-1 w-7 bg-primary mt-1" />
      <div className="h-1 w-7 bg-primary mt-1" />
      <div className="h-1 w-7 bg-primary mt-1" />
    </div>
  );
};

export default Burger;
