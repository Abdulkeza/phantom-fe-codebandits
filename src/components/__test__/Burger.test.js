import React from 'react';
import renderer from 'react-test-renderer';
import Burger from '../Burger.js';

describe('Burger', () => {
  it('should render Burger', () => {
    const elem = renderer.create(<Burger />).toJSON();
    expect(elem).toMatchSnapshot();
  });
});
