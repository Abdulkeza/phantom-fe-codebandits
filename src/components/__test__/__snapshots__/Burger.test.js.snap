// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Burger should render Burger 1`] = `
<div>
  <div
    className="h-1 w-7 bg-primary mt-1"
  />
  <div
    className="h-1 w-7 bg-primary mt-1"
  />
  <div
    className="h-1 w-7 bg-primary mt-1"
  />
</div>
`;
